.github/workflows/pylint.yml

```diff
@@ -0,0 +1,34 @@
+name: Python application
+
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade -r requirements.txt
+      - name: Lint with flake8
+        run: |
+          flake8 . --count --show-source --statistics
+      - name: Lint with pylint
+        run: |
+          pylint src tests
```

Comment:
```suggestion
          pylint src tests --rcfile=.pylintrc
```

Reply:
Not necessary

Reply:
I didn't know it will load it by default.